CHANGELOG.md

```diff
@@ -40,6 +40,8 @@
 
 ##### Bug Fixes
 
+* `CXComment.commandName()` was returning nil on custom code comments since xcode 8.1 sdk. This caused a force unwrap when generating documentation. Inline command comment is now used as a fallback to catch this edge case.
```

Comment:
This is in the wrong changelog section. An empty changelog section was added in 420b07a which should contain this entry. Try rebasing this PR to be able to move this entry to that section.